monitoring/ceph-mixin/dashboards_out/osds-overview.json

```diff
@@ -476,14 +474,14 @@
             {
                "expr": "count(ceph_bluefs_wal_total_bytes)",
                "format": "time_series",
-               "intervalFactor": 1,
+               "intervalFactor": 2,
```

Comment:
Why are we changing this?

Reply:
This was originally 2 only..corrected in this PR

Reply:
Ok thanks!